claim/README.md

```diff
@@ -167,11 +167,11 @@ Use these keys and the information below to troubleshoot the ACLK.
 
 If `cloud-enabled` is `false`, you probably ran the installer with `--disable-cloud` option.
 
-Additionally, check that the `netdata cloud` setting in `netdata.conf` is set to `enable`:
+Additionally, check that the `enabled` setting in `var/lib/netdata/cloud.d/cloud.conf` is set to `true`:
```

Comment:
```suggestion
Additionally, check that the `enabled` setting in `/var/lib/netdata/cloud.d/cloud.conf` is set to `true`:
```

Reply:
I didn't want to do that because it may be something like `/opt/netdata/var/lib/netdata/cloud.d/cloud.conf` and if I use a root path it may confuse the user.

Reply:
Ah, I see. Then no need to change it.